grails-app/views/supplier/show.gsp

```diff
@@ -0,0 +1,50 @@
+
+<%@ page import="org.pih.warehouse.core.Organization" %>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <meta name="layout" content="custom" />
+        <g:set var="entityName" value="${warehouse.message(code: 'location.supplier.label', default: 'Supplier')}" />
+        <title><warehouse:message code="default.show.label" args="[entityName]" /></title>
+    </head>
+    <body>
+        <div class="body">
+            <g:if test="${flash.message}">
+	            <div class="message">${flash.message}</div>
+            </g:if>
+            <div>
+                <div style="border-width: 5px">
+                    <h2>${supplier?.name}</h2>
+                    <g:each var="location" in="${supplier.locations.sort { it.name } }">
+                        <div style="margin: 10px 0 10px 30px;">
+                            <g:link controller="location" action="show" id="${location.id}">
+                                ⚫ ${location?.name}
```

Comment:
What's up with the blackhole?

Reply:
It's not needed here. I added it while testing so it was easier to count and check all locations.